libraries/rush-lib/src/logic/Git.ts

```diff
@@ -298,7 +298,7 @@ export class Git {
    * Gets the remote default branch that maps to the provided repository url.
    * This method is used by 'Rush change' to find the default remote branch to compare against.
    * If repository url is not provided or if there is no match, returns the default remote's
-   * default branch 'origin/master'.
+   * default branch 'origin/main'.
```

Comment:
Where is this implemented?

Reply:
https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/src/api/RushConfiguration.ts#L46